plugin.audio.rautemusik/resources/language/English/strings.xml

```diff
@@ -2,65 +2,4 @@
 <strings>
 	<string id="30001">General</string>
```

Comment:
It would be nice if you could move the language files to the new .po format at some point in the future. Not a blocker for merging this PR though.

Reply:
Thank you! I will schedule this for the next release of the plugin.